### PR TITLE
Touch up go mod instructions in install

### PR DESCRIPTION
**Pull request by TomSweeneyRedHat**
Add steps on how to use your own fork to vendor in code using go mod
to the install instructions.

Signed-off-by: TomSweeneyRedHat <tsweeney@redhat.com>

### CODE REVIEW

**Comment by TomSweeneyRedHat** [comment]
@vrothberg PTAL

File: install.md [hunk 1]
```diff
@@ -385,3 +385,14 @@ Buildah uses Go Modules for vendoring purposes.  If you need to update or add a
...
+
+### Vendor from your own fork
+
+If you wish to vendor in your personal fork to try changes out (assuming containers/storage in the below example):
+
+ * `go mod edit -replace github.com/containers/storage=github.com/{mygithub_username}/storage@YOUR_BRANCH`
+ * `make vendor`
+ 
+To revert
+ * `go mod edit -dropreplace`
```
Comment (vrothberg) [inline install.md]: [...]  `-dropreplace github.com/containers/storage` (to avoid confusion)

Comment (TomSweeneyRedHat) [reply install.md]: fixed, thx!

**Comment by rhatdan** [comment]
@rh-atomic-bot r+

**Pull request closed without merging**